records_mover/session.py

```diff
@@ -115,7 +115,8 @@ def __init__(self,
 
     def get_default_db_engine(self) -> Engine:
         if self._default_db_creds_name is None:
-            return db_engine(self)
+            db_facts = db_facts_from_env()
+            return engine_from_db_facts(db_facts)
```

Comment:
The function was two lines long and only used in one place - let's just fold it in here.